docs/wiki/archive/GPS-Rescue-v4-5.md

```diff
@@ -0,0 +1,526 @@
+- New CLI settings to fine-tune the disarm threshold, adjust the smoothing of the velocity D factor, and adjust how aggressively the IMU will handle a heading IMU error
+- Some CLI settings have been re-named to be more consistent.
+- A new `GPS_CONNECTION` debug to help resolve technical issues relating to the connection with the module and the CPU usage of the GPS code
+- the Status CLI menu now reports the status of the connection to the GPS unit, including baud rate and the detected type of module
+- Numerous bugfixes and tweaks for both safety and performance
+
+With these changes, GPS Rescue is now far more reliable than ever before.
+
+:::caution
+The user **must** configure the Stage 1 settings correctly, as described below, or the quad will drop and may crash in a true signal loss failsafe.
+:::
+
+For technical information about changes since 4.4, please see:
+- [PR12799, module hardware support](https://github.com/betaflight/betaflight/pull/12799): thanks to unit (Freasy), ctznooze, zzyzx and ledvinap
+- [PR12900, IMU heading fixes, bugfixes](https://github.com/betaflight/betaflight/pull/12900): thanks to ctznooze, ledvinap
+- [PR12859, IMU Yaw CoG gain](https://github.com/betaflight/betaflight/pull/12792]: thanks to ledvinap
+
+
+## About GPS Rescue
+
+GPS Rescue is an advanced safety feature that can autonomously fly the craft home, after loss of the RC or video link, until it is close enough to re-gain control. It can also land and disarm with reasonable precision.
+
+When activated, the craft should, sequentially:
+
+- enable level mode and climb to a user-configured altitude, at a configurable rate,
+- rotate at 180 deg/s until pointing towards home,
+- pitch forward and fly towards home at a user-configurable speed (note: the nose of the quad will jump up and down in pitch during the rescue in order to keep the forward speed accurately controlled),
+- correct the attitude of the quad to keep it pointing to home if it drifts off-course due to wind, and will mix in some roll to minimise lateral drift
+- when at a user-defined distance from home, start to descend at a configurable rate,
+- reduce forward velocity as it approaches home, and reduce descent rate as it approaches ground
+- enable impact detection when below the set landing altitude
+- automatically disarm on touching down
+
+Typical applications include:
+
+- automatically and autonomously flying home when the radio link is lost (requires failsafe stage 2 to be set to GPS Rescue)
+- climbing and flying back on a lost FPV video link is restored (requires manual mode switch activation by the pilot)
+- a 'panic' switch for LOS pilots who lose orientation (requires manual mode switch activation by the pilot)
+
+## Important:
+
+:::danger
+
+- Only use the 10.10 Configurator GUI. For earlier Configurator versions, only make changes to GPS Rescue parameters in the CLI!
```

Comment:
I would change Configurador versión namimg to 10.10.x

Reply:
Thanks, will do